guide-us-english.tex

```diff
@@ -329,7 +329,7 @@ \subsection{Strategic Vision}
 	\item clearly align the entire organization along a shared path forward.
 	\item compellingly articulate why the organization exists.
 	\item describe what the organization will do to leverage key assets in support of its mission.
-	\item respond to rapidly changing market conditions.
+	\item respond rapidly to changing market conditions.
```

Comment:
Disagree, because the intent of the item changes. It meant to refer to the outside world changing rapidly, the commit removes that point and makes it about responding rapidly internally. You could even make a case for "respond rapidly to rapidly changing market conditions", but that's really ugly :-).